### PR TITLE
Fixed #26975 -- Clarified how Django looks for fixture files.

### PR DESCRIPTION
**Pull request by filiplajszczak**
Explain the different effects of relative and absolute paths on `loaddata`. 

Suggest that fixture files be namespaced, which makes the documentation for fixtures consistent with that for templates and static files. 

Based on previous proposal from PR #6996 by @brotskydotcom  made more howto-like and less reference-like.

### CODE REVIEW

File: docs/howto/initial-data.txt [hunk 1]
```diff
@@ -80,16 +80,32 @@ from the fixture and reloaded into the database. Note this means that if you
 change one of the rows created by a fixture and then run :djadmin:`loaddata`
 again, you'll wipe out any changes you've made.
 
-Where Django finds fixture files
---------------------------------
+Telling Django where to look for fixture files
```
Comment (evildmp) [inline docs/howto/initial-data.txt:83]: I would suggest: _How to tell Django..._ or maybe _Tell Django..._ (verbs are good for how-to guides).

File: docs/howto/initial-data.txt [hunk 2]
```diff
@@ -80,16 +80,32 @@ from the fixture and reloaded into the database. Note this means that if you
...
+fixtures, so the command ``loaddata sample`` would find the file
+``my_app/fixtures/sample.json``.  This works with relative paths as
+well, so ``loaddata my_app/sample`` will find the file
+``my_app/fixtures/my_app/sample.json``.
+
+You can provide additional directories with your fixtures.  Django will also
+look for fixtures in the list of directories provided in the
+:setting:`FIXTURE_DIRS` setting.
+
+If you want to completely prevent default search form happening, use an absolute
```
Comment (evildmp) [inline docs/howto/initial-data.txt:96]: "To completely prevent..."

File: docs/howto/initial-data.txt [hunk 3]
```diff
@@ -80,16 +80,32 @@ from the fixture and reloaded into the database. Note this means that if you
...
+path to specify the location of your fixture file, as in
+``loaddata /path/to/sample``.
+
+.. admonition:: Fixture file namespacing
```
Comment (evildmp) [inline docs/howto/initial-data.txt:100]: "Namespace your fixture files"

**Review by evildmp — COMMENTED** [review]
This looks great!

File: docs/howto/initial-data.txt (hunk 1 above)
Comment (filiplajszczak) [reply docs/howto/initial-data.txt]: I like the second option more, we have "How to" in the title of the whole guide.

**Review by evildmp — APPROVED** [review]
These additional changes look good to me.

**Comment by felixxm** [comment]
@filiplajszczak Thanks for this patch :+1: Welcome aboard :boat: 

I pushed tiny edits.

**Comment by filiplajszczak** [comment]
@felixxm Commit message is definitely better now. I was  hesitant about `Co-authored-by:` as I was not sure if the author of the original PR would like it. That's why I made it more vague, but If you fill that I was too scrupulous, let it be.

**Comment by clickonetwo** [comment]
@filiplajszczak I much appreciate that you picked this up and fixed it; I got busy at work (a long time ago) but always felt bad that I never got back to it.  I'm very appreciative of the co-author credit.

cc @felixxm @evildmp

**Comment by brotskydotcom** [comment]
@filiplajszczak I much appreciate that you picked this up and fixed it; I got busy at work (a long time ago) but always felt bad that I never got back to it. I'm very appreciative of the co-author credit.

cc @felixxm @evildmp

**Pull request merged**